data/energy_totals_XX_2038.csv

```diff
@@ -0,0 +1,2 @@
+,agriculture biomass,agriculture electricity,agriculture oil,electricity rail,electricity residential,residential biomass,residential gas,residential heat biomass,residential heat gas,residential heat oil,residential oil,services biomass,services electricity,services gas,services oil,total domestic aviation,total domestic navigation,total international aviation,total international navigation,total rail,total residential space,total residential water,total road,total services space,total services water,electricity residential space,electricity residential water,district heat share,electricity services space,electricity services water
```

Comment:
Is this file intended to be here? This is 2038 specific output, but not required as input I think.